chalice/app.py

```diff
@@ -350,6 +359,13 @@ def __eq__(self, other):
         return False
 
 
+class File(object):
+    def __init__(self, name, content):
+        self.name = name
+        self.content = content
+        self.size = len(content)
```

Comment:
Can we add a repr to this?  We should at least include the filename (and maybe content type) to help with debugging and test failures.